### PR TITLE
Added: homepage, landingpage

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
Files to check: 
Pages:
  -homepage
  -landingpage
  -mainpage
  
Components:
  -clock
  -quote
  -wether

image

image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  border: none;
+  border-bottom: 2px solid white;
+  background-color: transparent;
+}
+
+input:focus{
+  color: white;
+  /* font-size: 2rem;
+  font-weight: 400; */
+}
```
Comment (theVivek-kumar) [inline src/App.css:33]: remove comments

Comment (theVivek-kumar) [inline src/App.css:33]: here as well remove comment

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  justify-content: center;
+  align-items: center;
+}
+
+.time-container{
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 1050%;
```
Comment (theVivek-kumar) [inline src/App.css:46]: fix this

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    letter-spacing: -5px;
+    font-weight: 520;
```
Comment (theVivek-kumar) [inline src/App.css]: it  should in multiple of 100s

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+.weather-input>button{
+  background-color: transparent;
+  color: white;
+  font-size: 0.8rem;
+  font-weight: 200;
+  margin: 0.2rem;
+  padding:0.3rem;
+  transition:1s; 
+
```
Comment (theVivek-kumar) [inline src/App.css:161]: css is well written

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import './App.css';
```
Comment (theVivek-kumar) [inline src/App.js:1]: put this in  a separate index.js file then refer that file here that will  make ur code cleaner

File: src/components/quoteComponent/Quote.jsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+import { React , useState, useEffect } from 'react'
+import './quote.css'
+
+function Quote() {
+    const [quote, setQuote] = useState('')
+    const [author, setAuthor] = useState('')
+
+    const getQuote = () => {
+        fetch('https://api.quotable.io/random')
+        .then(res => res.json())
```
Comment (theVivek-kumar) [inline src/components/quoteComponent/Quote.jsx:11]: remove console.log

File: src/pages/homePage/HomePage.jsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
+
+import { React, useState} from 'react'
+import Clock from '../../components/clock/Clock'
+import Quote from '../../components/quoteComponent/Quote'
+
+function HomePage() {
```
Comment (theVivek-kumar) [inline src/pages/homePage/HomePage.jsx:6]: use reducer

File: src/pages/mainPage/MainPage.jsx [hunk 8]
```diff
@@ -0,0 +1,19 @@
+
+import React from 'react'
```
Comment (theVivek-kumar) [inline src/pages/mainPage/MainPage.jsx:2]: looks good to me  can merge

**Review by theVivek-kumar — APPROVED** [review]

File: src/App.css [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  position: relative;
+  padding: 1rem;
+}
+
+.weather-input>p{
+  font-size: 1rem;
+  height: 1rem !important;
```
Comment (JuvitaSaini09) [inline src/App.css]: suggestion:important should be avoided

File: src/App.js [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+import React from 'react';
+import Clock from './components/clock/Clock';
+import Weather from './components/weather/Weather';
+import LandingPage from './pages/landingPage/LandingPage';
+import { Route, Routes } from "react-router-dom";
+import MainPage from './pages/mainPage/MainPage';
+import HomePage from './pages/homePage/HomePage';
```
Comment (JuvitaSaini09) [inline src/App.js]: suggestion:you could have imported all pages one file

File: src/App.js [hunk 11]
```diff
@@ -0,0 +1,31 @@
+import './App.css';
+import React from 'react';
+import Clock from './components/clock/Clock';
+import Weather from './components/weather/Weather';
```
Comment (JuvitaSaini09) [inline src/App.js]: suggestion:you can import all components in one file

File: src/App.js [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+import Quote from './components/quoteComponent/Quote';
+
+function App() {
+  return (
+    <div className="App">
+     <div style={{ 
+  backgroundImage: `url("https://picsum.photos/1600/1000?random")`,
+  backgroundRepeat: 'no-repeat',
+  width:'100%',
+  height:'100vh',
```
Comment (JuvitaSaini09) [inline src/App.js]: inline styling should be avoided

**Review by JuvitaSaini09 — APPROVED** [review]
i have suggested some changes which you can make later,your code looks good to merge

File: src/App.js [hunk 13]
```diff
@@ -0,0 +1,31 @@
+import './App.css';
+import React from 'react';
+import Clock from './components/clock/Clock';
+import Weather from './components/weather/Weather';
+import LandingPage from './pages/landingPage/LandingPage';
+import { Route, Routes } from "react-router-dom";
+import MainPage from './pages/mainPage/MainPage';
+import HomePage from './pages/homePage/HomePage';
+import Quote from './components/quoteComponent/Quote';
```
Comment (yaswanthmyneni) [inline src/App.js]: suggestion: export all components from index.js in components folder and import from it

File: src/App.js (hunk 12 above)
Comment (yaswanthmyneni) [inline src/App.js]: changes: use className instead of styling inline.

File: src/components/weather/Weather.jsx [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+    .then((res) => {
+      console.log("response", res.data) 
+      setData(res.data)
+    }).catch((err) => {
+      console.log("err", err)
+    })
+  }
+
+  const handleChangeInput = (e) => {
+    console.log("value", e.target.value)
```
Comment (yaswanthmyneni) [inline src/components/weather/Weather.jsx]: changes: remove console.log

File: src/index.js [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+    <BrowserRouter>
+    <App />
+    </BrowserRouter>
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
```
Comment (yaswanthmyneni) [inline src/index.js:19]: changes: remove commented code

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes

File: src/App.css (hunk 3 above)
Comment (Adarsh-Patel-Dev) [reply src/App.css]: it is multiple of 100

File: src/App.css (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply src/App.css:33]: ok

**Pull request merged**